tests/api/enqueue.cpp

```diff
@@ -137,3 +137,81 @@ TEST_F(WithCommandQueue, WorkDim) {
     EnqueueUnmapMemObject(buffer, data);
     Finish();
 }
+
+TEST_F(WithCommandQueue, PodUBO) {
+    static const char* program_source =
+        "kernel void test(global int* out, int a, int4 b, int c) { *out = a + "
+        "b.x + c; }";
```

Comment:
Maybe using another component than `x` would make for a slightly better test?

Reply:
I'll use different components in each test.